src/seedu/addressbook/AddressBook.java

```diff
@@ -211,13 +211,17 @@ public static void main(String[] args) {
         processProgramArgs(args);
         loadDataFromStorage();
         while (true) {
-            String userCommand = getUserInput();
-            echoUserCommand(userCommand);
-            String feedback = executeCommand(userCommand);
-            showResultToUser(feedback);
+            processUserCommand();
         }
     }
 
+	private static void processUserCommand() {
+		String userCommand = getUserInput();
+		echoUserCommand(userCommand);
```

Comment:
Seems like there is an Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.